app/assets/json/translations/react/en.json

```diff
@@ -1,174 +1,217 @@
-  "signIn.signedOutSuccessfully": "You have successfully logged out of your account.",
-  "signIn.userTokenValidationTimeout": "We care about your security. We logged you out due to a connection issue. Please sign in to continue.",
-  "t.available": "available",
-  "t.child": "child",
-  "t.children": "children",
-  "t.close": "Close",
-  "t.continue": "Continue",
-  "t.dahliaSanFranciscoHousingPortal": "DAHLIA San Francisco Housing Portal",
-  "t.homePage": "Home page",
-  "t.income": "income",
-  "t.incomeRange": "Income Range",
-  "t.menu": "Menu",
-  "t.minimumIncome": "Minimum Income",
-  "t.numberRange": "%{minValue} to %{maxValue}",
-  "t.perMonth": "per month",
-  "t.perYear": "per year",
-  "t.rent": "Rent",
-  "t.seeDetails": "See Details",
-  "t.skipToMainContent": "Skip to main content",
-  "t.to": "to",
-  "t.unitType": "Unit Type",
-  "t.units": "Units",
-  "t.waitlist": "Waitlist",
-  "welcome.newListingEmailAlert": "Get emailed whenever a new listing is posted.",
-  "welcome.seeRentalListings": "Rent",
-  "welcome.seeSaleListings": "Buy",
-  "welcome.signUpToday": "Sign Up today",
-  "welcome.title": "Apply for affordable housing"
+    "config.routePrefix": "",
+    "errorBoundary.description": "An application error occurred. Check back later.",
+    "errorBoundary.goHome": "Back to homepage",
+    "footer.cityCountyOfSf": "City & County of San Francisco",
+    "footer.contact": "Contact",
+    "footer.dahliaDescription": "DAHLIA: San Francisco Housing Portal is a project of the <br /><a className='text-white' href='%{mohcdUrl}' target='_blank'>Mayor's Office of Housing and Community Development</a>",
+    "footer.disclaimer": "Disclaimer",
+    "footer.forGeneralQuestions": "For general program inquiries, you may call MOHCD at 415-701-5500.",
+    "footer.forListingQuestions": "For listing and application questions, please contact the agent displayed on the listing.",
+    "footer.giveFeedback": "Give Feedback",
+    "footer.inPartnershipWith": "in partnership with<br /><a className='text-white' href='%{sfdsUrl}' target='_blank'>San Francisco Digital Services</a><br /><a className='text-white' href='%{mayorUrl}' target='_blank'>Mayor's Office of Civic Innovation</a>",
+    "footer.privacyPolicy": "Privacy Policy",
+    "idleTimeout.sessionExpired": "Your session has expired.",
+    "idleTimeout.sessionInactivity": "To protect your identity, your session will expire in one minute due to inactivity. You will lose any unsaved information you've entered into the application for this listing if you choose not to respond.",
+    "idleTimeout.sessionInactivityLoggedIn": "To protect your identity, your session will expire in one minute due to inactivity. You will lose any unsaved information and be logged out if you choose not to respond.",
+    "idleTimeout.stayLoggedIn": "Stay logged in?",
+    "label.clear": "Clear",
+    "label.editEligbility": "Edit Eligibility",
+    "label.read_more": "Read More",
```

Comment:
This string on prod is all lowercase `read more` - I personally think it looks better uppercase like this, just wanted to make sure this was intentional

Reply:
It was intentional, but you're right we should limit changes. I will revert to `read more`

In terms of camelCase vs snake_case...I wasn't sure if there was some magic happening to map angular keys to react so I was keeping the same keys, but I think you're also right here and we should remain consistent. Let's go with camel case fyi @jacksongonzales

Reply:
There's no angular-react mapping so we can name them whatever we want :)